common/app/model/facia/PressedCollection.scala

```diff
@@ -64,6 +66,17 @@ case class PressedCollection(
     copy(hasMore = hasMore)
   }
 
+  def fullDCR(visible: Int): PressedCollection = {
+    val liteCurated = curated.take(visible)
+    val liteBackfill = backfill.take(visible - liteCurated.length)
```

Comment:
What is `visible` and how is this defined? I feel this is quite important as I'm guessing there is some logic in Frontend deciding how many cards are shown based on the container type etc, is there a way we can avoid this logic being in frontend somehow? (Like passing the number of visible cards in the request from DCR, e.g `/*path/show-more/*id.json?dcr=true&visible=12`) 

I'm guessing in the future DCR will start to change the numbers of cards shown in different situations, etc.

----

I'm also a little bit confused at to what `lite` means? Is this the cards that are visible or the cards that aren't visible?